src/mpi4py/MPI/msgbuffer.pxi

```diff
@@ -81,14 +81,41 @@ cdef inline bint is_IN_PLACE(object obj):
 
 #------------------------------------------------------------------------------
 
+cdef inline int is_bigendian() nogil:
+    cdef int i = 1
+    return (<char*>&i)[0] == 0
+
+cdef inline const char *getformat(const char format[]) except NULL:
+    cdef char byteorder = format[0]
+    if byteorder == c'@': # native
+        return format + 1
+    if byteorder == c'<': # little-endian
```

Comment:
```suggestion
    elif byteorder == c'<': # little-endian
```

Reply:
Why? If any of the `if` branches is followed, the function either returns or raises exception.  So using successive `if` branches vs `if/elif` it is just as a matter of style with not semantic differences. Or am I wrong?

Reply:
Sorry, I don't remember what I was thinking earlier. 😞